erpnext/templates/pages/integrations/authorizenet_checkout.html

```diff
@@ -26,7 +26,7 @@ <h2 class="text-center">{{ payment_context.description }}</h2>
 						</div>
 						<div class="col-md-8 col-sm-6 pl-0 pr-0">
 							<input id="cardholder-name" name="cardholder-name" class="field w-100" placeholder="{{ _('John Doe') }}"
-								value="{{ payment_context.payer_name }}" />
+								value="" />
```

Comment:
Remove the value parameter.
You also need to remove the email id along with this

Reply:
removed value parameter from the name and email id field